### PR TITLE
Tickers

### PR DESCRIPTION
**Pull request by kolya182**

### CODE REVIEW

**Comment by kolya182** [comment]
@ArtemGoryunov WIll be working on `1`, `2` today. Regards `3` I think we need separate endpoint for token/coin info like social link, website, whitepaper etc. 
For `4` will be working over weekend and begging of next week.
So far next format used across all api's
```json
{
  "docs": [
    {
      
    }
  ]
```
}

**Comment by ArtemGoryunov** [comment]
Great, thank you!
Info like social link, website, whitepaper can be placed in to `/tickers/:id/info`.

**Comment by kolya182** [comment]
We have the endpoint for that kind of info already [](https://public.trustwalletapp.com/tokeninfo/ethereum/0xdd974d5c2e2928dea5f71b9825b8b646686bd200) I will docs. I'm my plan to write parsers to scrape all kind of info from the different source to make info even more full.

**Comment by kolya182** [comment]
Endpoint https://public.trustwalletapp.com/tokeninfo/ethereum/0xdd974d5c2e2928dea5f71b9825b8b646686bd200

File: src/controllers/AppCheck.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { sendJSONresponse } from "../common/Utils";
 
 export class AppCheck {
     public android(req: Request, res: Response) {
-        const version = "1.6.109"
-        const build = 282
+        const version = "1.6.112"
+        const build = 285
```
Comment (vikmeup) [inline src/controllers/AppCheck.ts]: is this needs to be changed in this PR?

File: src/controllers/Interfaces/Ticker.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    symbol: string;
+    total_supply: number;
+    volume_24h: number;
+}
+
+export enum Currencies {
+    USD = "USD",
+}
+
+export enum TickersSortCriteria {
```
Comment (vikmeup) [inline src/controllers/Interfaces/Ticker.ts]: avoid this! we don't need this now

**Pull request closed without merging**